core/src/main/java/io/parsingdata/metal/data/selection/ByPredicate.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2013-2016 Netherlands Forensic Institute
```

Comment:
Agreed. There's a separate issue however, because it's best if it's one big commit to fix this: #133.